content/docs/1.4.0/snapshots-and-backups/csi-snapshot-support/enable-csi-snapshot-support.md

```diff
@@ -25,15 +24,18 @@ Note that the snapshot controller YAML files mentioned below deploy into the `de
 > For example, on a vanilla Kubernetes cluster, update the namespace from `default` to `kube-system` prior to issuing the `kubectl create` command.
 
 Install the Snapshot CRDs:
-1. Download the files from https://github.com/kubernetes-csi/external-snapshotter/tree/release-4.0/client/config/crd
+1. Download the files from https://github.com/kubernetes-csi/external-snapshotter/tree/v5.0.1/client/config/crd
```

Comment:
Will it be helpful to mention installing a compatible version of the built-in external-snapshotter CSI sidecar?

Reply:
external-snapshotter CSI sidecar is shipped by Longhorn. Could you explain a bit more? Thank you

Reply:
Suggest explaining why the version mentioned is `v5.0.1` or a compatible one, this is because it's referenced from the version of the external-snappshotter used in Longhorn, right?

Reply:
Ah. I see. Thank you. Fixed